api/restapi.py

```diff
@@ -368,8 +368,8 @@ def add_social_account(self, request):
         try:
             p = Profile(self.db)
             if "account_type" in request.args and "username" in request.args and "proof" in request.args:
-                p.add_social_account(request.args["account_type"][0], request.args["username"][0],
-                                     request.args["proof"][0] if "proof" in request.args else None)
+                p.add_social_account(request.args["account_type"][0], request.args["username"][0].decode('utf8'),
+                                     request.args["proof"][0].decode('utf8') if "proof" in request.args else None)
```

Comment:
The guideline say nothing gets merged if it isn't PEP-8 compliant, and these lines are over 80 characters.

Nitpick: you're mixing single and double quotes.

Maybe this instead:
```Python
                if "proof" in request.args:
                    proof = request.args["proof"][0].decode("utf8")
                else:
                    proof = None
                p.add_social_account(
                    request.args["account_type"][0], 
                    request.args["username"][0].decode('utf8'),
                    proof,
                )
```